app/src/components/existing-request/existing-request-search.vue

```diff
@@ -76,12 +76,11 @@
               cols="1"
               class="max-width"
             >
-              <v-img
+              <img
```

Comment:
I changed this so it would work when deployed (eg, to Preview URL). A number of different src syntaxes worked locally but not deployed.